src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,10 @@
+package duke;
+
+public class DukeException extends Exception {
+    public void missingDescription(String command){
```

Comment:
According to the Java coding standard, there should be a whitespace between the brackets `){`.